tla/ccfraft.tla

```diff
@@ -1141,9 +1129,14 @@ Spec == Init /\ [][Next]_vars
 \* Correctness invariants
 \* These invariants should be true for all possible states
 
-\* Committed log entries should not conflict
+\* Committed log entries should never conflict between servers
+\* This is a key safety invariant and should always be checked
+\* LogInv checks for safety violations across space
```

Comment:
Perhaps, this statement can be better illustrated by stating it as a theorem:

```tla
THEOREM Spec => []LogInv

THEROEM Spec => CommittedLogNeverChangesProp
```

Reply:
Done in cad2f39a7d710959f76bef958c62dce182020c23.